src/Microsoft.AspNetCore.ResponseCompression/IResponseCompressionProvider.cs

```diff
@@ -23,5 +23,12 @@ public interface IResponseCompressionProvider
         /// <param name="context"></param>
         /// <returns></returns>
         bool ShouldCompressResponse(HttpContext context);
```

Comment:
Now that the flow has changed it may make sense to merge ShouldCompressResponse into GetCompressionProvider